src/guides/libraries/c-interop.md

```diff
@@ -266,3 +325,10 @@ to automatically create FFI wrappers from C header files.
 [mini tutorial.]: https://github.com/dart-lang/sdk/blob/main/samples/ffi/sqlite/docs/sqlite-tutorial.md
 [`NativeType`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/NativeType-class.html
 [ffigen]: {{site.pub-pkg}}/ffigen
+[`native_add_library`]: https://github.com/dart-lang/native/tree/main/pkgs/native_assets_cli/example/native_add_library
+[`native_add_app`]: https://github.com/dart-lang/native/tree/main/pkgs/native_assets_cli/example/native_add_app
+[`Native`]: {{site.dart-api}}/dev/dart-ffi/Native-class.html
+[`DefaultAsset`]: {{site.dart-api}}/dev/dart-ffi/DefaultAsset-class.html
```

Comment:
You link to the dev versions here. Is this intentional or are the 3.2.0 API docs sufficient? If not, can we find a happy medium with the beta versions? :)

Reply:
I think once the API docs are out for 3.2 stable, it's fine to link to stable

Reply:
They were published a bit earlier and looks like they are now the default. I've updated the links :D 

Thanks!